dragonfly/implementations/psyllid_provider.py

```diff
@@ -132,11 +135,10 @@ def reactivate(self, channel):
         if self.status_value_dict[channel] == 4:
             logger.info('Reactivating Psyllid instance of channel {}'.format(channel))
             self.provider.cmd(self.queue_dict[channel], 'reactivate-daq')
-            logger.info(result)
         else:
             logger.warning('Cannot reactivate Psyllid instance of channel {}'.format(channel))
             return False
-        time.sleep(1)
+        time.sleep(2)
```

Comment:
Are we using this sleep to ensure there's time for the cmd to be processed? Should that be done by the time that line 137 returns? We don't need to delay the PR for it, but I'm pretty nervous about needing to hardcode sleeps like this.

Reply:
We have to wait with checking psyllid's status until it has finished changing that. Otherwise the returned status will be activating or deactivating instead of activated.
In psyllid/source/control/daq_control, reactivate is simply deactivate followed by activate. Psyllid itself waits for a hardcoded second between those too calls because it would otherwise run into the same problem:

    void daq_control::reactivate()
    {
        deactivate();
        std::this_thread::sleep_for( std::chrono::seconds(1) );
        activate();
        return;
    }

We could not check the status after telling psyllid to deactivate, activate or reactivate. The status switching in psyllid seems pretty reliable by now.
Would you prefer that?

Reply:
Also:
The trigger and frequency settings are active node configurations. 
When psyllid is reactivated they are lost and have to be reset. You can only set them when psyllid is activated. With the hardcoded sleep we make sure that new active node settings cannot be applied while psyllid is still reactivating.

Reply:
I guess the point for the sleep is well made if that's the psyllid code. It isn't obvious to me that things are self-consistent, but that's maybe not a question for here.